docs/usage/ipv6.md

```diff
@@ -50,19 +34,23 @@ When provisioning a DualStack cluster, the GCP provider creates distinct subnets
 
 - This subnet is dedicated to IPv6 services. It is created due to GCP's limitation of not supporting IPv6 reservation for services.
 
+### 3. **Internal Subnet (optional)**
+
+- This subnet is dedicated for internal load balancer. Currently are only internal IPv4 loadbalancer supported. They are provisioned by the Cloud Controller Manager (CCM).
+
 ---
 
 ## Additional Components
 
 ### 1. **Ingress-GCE**
```

Comment:
Let's combine the headers. There is no need for the additional structure and/or a numbering with just one entity.

Reply:
Maybe it makes even more sense to move the content to the Load Balancer section.